workflow-service/src/main/resources/application-local.yml

```diff
@@ -20,16 +20,3 @@ spring:
 springdoc:
   writer-with-order-by-keys: true
   writer-with-default-pretty-printer: true
-management:
```

Comment:
Why do we want to remove this here and for dev profile?

Reply:
It's already part of the application.yaml, and it's not needed at all.